.babelrc.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+  "plugins": [process.env.BABEL_ENV === 'umd' && "@babel/plugin-transform-runtime"].filter(Boolean),
```

Comment:
I would make this part a bit clearer by using a ternary or something like that :)

```suggestion
  "plugins": (process.env.BABEL_ENV === 'umd')
    ? ["@babel/plugin-transform-runtime"]
    : [],
```